src/pages/lib-v1/auth/switch-auth/q/platform/[platform].mdx

```diff
@@ -3,541 +3,107 @@ export const meta = {
   description: `Learn more about how to switch between different auth flows in Amplify.`
 };
 
-## Identity Pool Federation
+import federation from '/src/fragments/lib-v1/v6/callout/auth-federation.mdx';
 
-V6_TODO: Remove this whole section and replace with the credentials provider.
+<Fragments fragments={{ js: federation }} />
 
-You can alternatively use `Auth.federatedSignIn()` to get AWS credentials directly from Cognito Federated Identities and not use User Pool federation. If you have logged in with `Auth.signIn()` you **can not** call  `Auth.federatedSignIn()` as Amplify will perform this federation automatically for you in the background.
+For client side authentication there are three different flows:
 
-In general, if you are using Cognito User Pools to manage user Sign-Up and Sign-In, you should only call `Auth.federatedSignIn()` when using OAuth flows or the Hosted UI.
+1. `USER_SRP_AUTH`: The `USER_SRP_AUTH` flow uses the [SRP protocol (Secure Remote Password)](https://en.wikipedia.org/wiki/Secure_Remote_Password_protocol) where the password never leaves the client and is unknown to the server. This is the recommended flow and is used by default.
 
-```js
-import { Auth } from 'aws-amplify';
+2. `USER_PASSWORD_AUTH`: The `USER_PASSWORD_AUTH` flow will send user credentials unencrypted to the backend. If you want to migrate users to Cognito using the "Migration" trigger and avoid forcing users to reset their passwords, you will need to use this authentication type because the Lambda function invoked by the trigger needs to verify the supplied credentials.
```

Comment:
I will not say unencrypted because that is not correct, is encrypted by SSL protocol

Reply:
```suggestion
2. `USER_PASSWORD_AUTH`: The `USER_PASSWORD_AUTH` flow will send user credentials to the backend. This is needed in order to migrate users to a different Cognito User Pool.
```